core/src/main/scala/scalacss/package.scala

```diff
@@ -119,6 +119,8 @@ package object scalacss {
     /** Value to be applied to a HTML element's `class` attribute. */
     val htmlClass: String =
       (className.value /: addClassNames)(_ + " " + _.value)
+
+    def +(s : StyleA)(implicit c: Compose): StyleA = StyleA(className, Vector(s.className) ++ addClassNames ++ s.addClassNames, style.compose(s.style))
```

Comment:
Add newline after `=` and can use `+:` instead of `Vector(_) ++`. Like this:
```scala
def +(s : StyleA)(implicit c: Compose): StyleA =
  StyleA(className, s.className +: addClassNames ++ s.addClassNames, style.compose(s.style))
```